pom.xml

```diff
@@ -27,6 +27,12 @@
 		    <artifactId>slf4j-api</artifactId>
 		    <version>1.7.21</version>
 		</dependency>
+		
+		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-log4j12</artifactId>
+			<version>1.7.21</version>
+        </dependency>
```

Comment:
Ik weet eerlijk gezegd niet of dit wel de bedoeling is. Volgens mij krijgen we hiermee straks errors als we de tool in de Docker container proberen te draaien want die levert z'n eigen logger. Zie ook https://github.com/topicusonderwijs/mssql2monetdb/blob/master/docker/Dockerfile